images/ci-kubernetes-e2e-kubeadm/Dockerfile

```diff
@@ -0,0 +1,63 @@
+# Copyright 2017 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM gcr.io/k8s-testimages/kubekins-e2e:v20170104-9031f1d
+MAINTAINER beacham@google.com
+
+RUN apt-get update && apt-get install -y \
+    build-essential \
+    openjdk-7-jdk \
+    pkg-config \
+    zip \
+    unzip \
+    zlib1g-dev \
+    bash-completion \
+    git \
+    wget \
+    python \
+    python-pip && \
+    apt-get clean
+
+ENV GCLOUD_VERSION 138.0.0
+RUN cd / && \
+    wget https://dl.google.com/dl/cloudsdk/channels/rapid/downloads/google-cloud-sdk-$GCLOUD_VERSION-linux-x86_64.tar.gz && \
+    tar xf google-cloud-sdk-$GCLOUD_VERSION-linux-x86_64.tar.gz && \
+    rm google-cloud-sdk-$GCLOUD_VERSION-linux-x86_64.tar.gz && \
+    ./google-cloud-sdk/install.sh
+ENV PATH "/google-cloud-sdk/bin:${PATH}"
+
+# Add kubernetes-anywhere dependencies: jsonnet, terraform
+ENV TERRAFORM_VERSION 0.7.2
+
+RUN apt-get update && \
+    apt-get install -y unzip
```

Comment:
Oh it's worse than that; this section was completely redundant given the other apt-get block (each section came from a different source Dockerfile I was borrowing from). I removed these entirely.